source/test/F0.Minesweeper.Logic.Tests/MinefieldDifficultyTest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace F0.Minesweeper.Logic.Tests
 {
-	public class MinefieldOptionsTest
+	public class MinefieldDifficultyTest
```

Comment:
this can now be also named back to `MinefieldOptionsTest`

Reply:
Right! Done.